neqo-transport/src/connection/tests/ecn.rs

```diff
@@ -0,0 +1,77 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
```

Comment:
I'm not seeing any CE markings being applied, to test CC response.  Is that coming later?

Reply:
I can extend it in https://github.com/mozilla/neqo/pull/1689.